### DISCUSSION TITLE
personal-development-mehr-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#25] [Announcements]
# personal-development-mehr-1402/

به سبک و سیاقی که از پارسال شروع کردم، گزارش توسعه فردی ماهانه‌ام رو منتشر می‌کنم.
کتاب‌ها کتاب ژن‌خودخواه رو تا حد زیادی خواندم و قسمت‌های کوچیکی‌اش مونده که می‌گذارم بعداً شاید بخونم. به نظرم ریچارد داوکینز (نویسنده کتاب) حرف کم آورده و دیگه حرفاش تکراری شده.
کتاب تفکر کند و سریع هم یه بخش‌هایی ازش خوندم. جالب بود.
کتاب پیچیدگی ملانی میچل رو شروع کردم. اول ترجمه‌اش رو گرفته بودم که خب یک افتضاح به تمام عیار بود و بعدش گفتم ولش کن.

https://aprd.ir/personal-development-mehr-1402/

**Comment by MortezaAfroozeh** [comment]
انشالله سربازی تون به سلامت😁❤
شدیدا منتظرم کم کم از سربازی هم بنویسید.

**Reply by pourmand1376** [reply]
سلامت باشید. چشم حتماً. فعلا روز اولی هست که گذاشتند بیایم خونه. 

ولی تو این چند روز خیلی مطلب برای خودم نوشتم. باید فکر کنم قسمت‌های قابل انتشارش رو بنویسم.
